### PR TITLE
Feature migration cleanup

### PR DESCRIPTION
**Pull request by roigcarlo**
An initial PR with some git basics

### CODE REVIEW

**Comment by farrufat-cimne** [comment]
Will this cause any conflict if other users already put this folders/files on the gitignore list?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Compiled python
+*.pyc
+
+# CMake directory
+cmake_build/*
+!cmake_build/example_configure.bat.do_not_touch
```
Comment (pooyan-dadvand) [inline .gitignore:6]: we should ignore the history of configure.xxx.do_not_touch?

**Comment by roigcarlo** [comment]
@farrufat-cimne  Since there were no other centralized .gitignore it is probable. But this is only a generic one. Feel free to add any other ignores here

**Comment by pooyan-dadvand** [comment]
Instead of a branch with a generic clean up purpose I would suggest smaller one with more specific purposes.

File: .gitignore (hunk 1 above)
Comment (roigcarlo) [reply .gitignore:6]: We should not.  Notice the ```!```
Basically I want to ignore everything there except those files

**Comment by roigcarlo** [comment]
Ok, maybe the name was misleading...

File: .gitignore (hunk 1 above)
Comment (pooyan-dadvand) [reply .gitignore:6]: Sorry, I have not notice that!

**Review by pooyan-dadvand — APPROVED** [review]
👍

**Pull request merged**